lnbits/templates/components.vue

```diff
@@ -1302,6 +1302,9 @@
         </div>
       </q-form>
     </q-card-section>
+    <q-card-section v-else-if="!allowed_new_users && authAction === 'register'">
+      <h5 class="text-center" v-text="$t('new_user_not_allowed')"></h5>
+    </q-card-section>
```

Comment:
can we just hide the register button and don't even show the `new_user_not_allowed` card section. i think it would be cleaner.

Reply:
ok, now by default we don't show the register card, if register is not allowed. I kept it in the markup, just in case someone, somehow ends up there!